docs/admin/access.rst

```diff
@@ -76,37 +80,38 @@ Project access control
 
 .. note::
 
-    By enabling ACL, all users are prohibited from accessing anything within a given
-    project, unless you add the permissions for them to do just that.
+    By turning on access-control lists, all users are prohibited from accessing anything
+    within a given project, unless you add the permissions for them to do just that.
 
 .. note::
 
-    This feature is not available for projects running the Hosted Libre plan.
+    This feature is unavailable for Libre plan projects on Hosted Weblate.
 
-You can limit user's access to individual projects. This feature is turned on by
+Limit user's access to individual projects by turning on
 :guilabel:`Access control` in the configuration of each respective project.
-This automatically creates several groups for this project, see :ref:`groups`.
+This automatically creates several groups for the project in question, see :ref:`groups`.
 
-The following choices exist for :guilabel:`Access control`:
+:guilabel:`Access control` can be set to:
 
 Public
     Publicly visible and translatable
 Protected
     Publicly visible, but translatable only for selected users
 Private
-    Visible and translatable only for selected users
+    Only visible and translatable for selected users
 Custom
-    Weblate does not manage users, see :ref:`custom-acl`.
+    Django manages users instead of Weblate, see :ref:`custom-acl`.
 
 .. image:: /images/project-access.png
 
-To allow access to this project, you have to add the privilege either
+Grant access to a project by adding the privilege either
 directly to the given user, or group of users in the Django admin interface,
```

Comment:
In my head it should be "Django admin-interface" to avoid the notion that it is administration of Django.

Reply:
good point